classes/class-gf-securesubmit.php

```diff
@@ -714,7 +716,7 @@ public function maybe_validate($validationResult) {
                 continue;
             }
 
-            if (empty($this->validateACH()) && $this->getSecureSubmitJsError() && $this->hasPayment($validationResult)) {
+            if (false == $this->validateACH() && $this->getSecureSubmitJsError() && $this->hasPayment($validationResult)) {
```

Comment:
I want to test against empty values as well, which is why I used double equals instead of triple